packages/liquidator/src/liquidationStrategy.js

```diff
@@ -115,17 +115,29 @@ module.exports = (
   }
 
   // Call this with only undercollateralized position
-  // SyntheticTokenBalance should be passed in respecting maxTokensToLiquidate
+  // SyntheticTokenBalance should be passed in respecting maxTokensToLiquidate.
+  // Responsible for emitting customized logs since its already parsing
+  // whether a liquidation is valid to send and for how many tokens.
   function processPosition({
     position,
     syntheticTokenBalance,
     currentBlockTime,
     maxCollateralPerToken,
     // mandated token to liquidate amount, an override
     maxTokensToLiquidateWei,
+    shouldEmitWDFSkipLogs,
```

Comment:
not a big fan of injecting a logging flag into this function, would rather like to see if this could be handled in the event handler in liquidator. i realize though this may be the best way as it is

edit: saw this is based on `blockWindowHasRequestedWithdraw`.  i would actually be ok with this  name as it represents how the boolean is calculated. its not logging specific, and theoretically could be used as additional info to process position.

Reply:
Let's just make this `debug` only for now